### PR TITLE
Referees 2018/generic files

### PR DESCRIPTION
**Pull request by idanstark42**
In order to give people a better control over the files they can choose to play and when to play them at the background of the clock. See readme part named "Selecting mp3 files" for full explanation.

### CODE REVIEW

**Comment by rikkertkoppes** [comment]
I won't merge it like this, although the idea is good.

Clock is a very simple application that should work by just opening the index.html from a file system, without any server.

In fact, it is used as such in schools, sports clubs and also as a timer for FLL judging. Your changes makes it all dependent on a server. This is not a bad thing, as long as it does not break the ability to work from a file system, which it does.

So, in the very least, the $http communication should be more robust and degrade gracefully. I can also see a road in which a custom mp3 is included in the mp3 folder and specified in the config file.

**Comment by rikkertkoppes** [comment]
There are also some general problems with the implementation

- the file `track.mp3` needs to be renamed to `track_150.mp3` for the default settings to work. This is not user friendly as it has to be renamed every time you change the countdown time (I like the idea of having a generic name for the start and end `track_start.mp3` and `track_stop.mp3` as well as optional time-bound files, for example a special extra drum roll at half time or something)
- the audio does not resume when pausing and resuming the clock
- the stop track only seems to play for 1 second, rather than its full length (7 seconds)

**Comment by rikkertkoppes** [comment]
Also, maybe you can have a look at #13

File: readme.md [hunk 1]
```diff
@@ -75,6 +75,18 @@ The following is a command line example on the windows command prompt, which wil
 This one will move the clock 10 px in the horizontal direction (to the right)
 - `mclient -n clock -t clock:nudge -d "{ ""direction"": ""x"",""amount"":"10" }"`
 
+### Selecting mp3 files
+In order to get to chose your own background sounds to play during the clock's run,
```
Comment (rikkertkoppes) [inline readme.md]: spelling: choose

**Comment by idanstark42** [comment]
Moved to [PR #20 ](https://github.com/FirstLegoLeague/clock/pull/20)

**Pull request closed without merging**